ister.py

```diff
@@ -73,7 +73,11 @@ def run_command(cmd, raise_exception=True, log_output=True, environ=None,
     try:
         LOG.debug("Running command {0}".format(cmd))
         sys.stdout.flush()
-        proc = subprocess.Popen(cmd.split(" "),
+        if shell:
+            full_cmd = ["sh", "-c", cmd]
```

Comment:
is this any different than just adding `shell=True` to the `Popen` call?

Reply:
agree would be more clean
```
subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=environ, shell=True)
```

Reply:
personal i would drop the extra shell argument and just use a shell by default... the installer worked fine when i did that.

Reply:
I agree with @bryteise that shell would not be good to use everywhere, but I think it makes sense for these scripts and commands.